nucliadb/nucliadb/search/api/v1/chat.py

```diff
@@ -63,16 +75,16 @@
 @version(1)
 async def chat_knowledgebox_endpoint(
     request: Request,
-    response: Response,
     kbid: str,
     item: ChatRequest = Body(examples=CHAT_EXAMPLES),
     x_ndb_client: NucliaDBClientType = Header(NucliaDBClientType.API),
     x_nucliadb_user: str = Header(""),
     x_forwarded_for: str = Header(""),
-) -> Union[StreamingResponse, HTTPClientError]:
+    x_synchronous: bool = Header(False),
```

Comment:
Can you add a description to the header? then we have it in the docs automatically (at least in the openapi spec)